pkg/sql/catalog/lease/lease_test.go

```diff
@@ -1932,12 +1931,8 @@ CREATE TABLE t.test2 ();
 
 	// Check that lease acquisition happens independent of lease being requested.
 	testutils.SucceedsSoon(t, func() error {
-		if count := atomic.LoadInt32(&testAcquiredCount); count <= 4 {
-			return errors.Errorf("expected more than 4 leases to be acquired, but acquired %d times", count)
-		}
-		released := releasedLeases()
-		if notYetReleased := expected.Difference(released); notYetReleased.Len() != 0 {
-			return errors.Errorf("expected %v to be released, released %v", expected.Ordered(), released.Ordered())
+		if count := atomic.LoadInt32(&testAcquiredCount); count <= 2 {
+			return errors.Errorf("expected more than 2 leases to be acquired, but acquired %d times", count)
```

Comment:
Do you mind explaining/justifying why this is the expected behavior under the new leasing model?

Reply:
note: discussed offline. this count should be exactly 2 because of the initial acquire on t1 and initial acquire on t2. 

This test is testing about the background renewal goroutine, but it's irreverent under the new model because we don't do renewal anymore in the new model, and therefore maybe we should rewrite/refactor this test to test the new behavior (i.e. renewal on the sqllliveness table, which affects the expiry of the leases)